### ISSUE TITLE
Font size of rows with submenu is not changeable

### ISSUE THREAD

**Issue by gson88** [matryer/xbar#663]
Hi.

It is no longer possible to change font-size or color of rows that contain a submenu.
Even though I set the size to 10, the rows with submenus look like they have size ~14.

Example output:
```
IOTA: $1.35|color="white" size="10"
--Qty: 1|color="white" size="10"
-----
--Total: $1|color="green" size="10"
1h:   -1.20%|color="red" size="10"
24h: -12.23%|color="red" size="10"
---
HBAR: $0.3014|color="white" size="10"
--Qty: 1|color="white" size="10"
-----
--Total: $1|color="green" size="10"
1h:   -0.76%|color="red" size="10"
24h: -5.80%|color="red" size="10"
```

Result:

![image](https://user-images.githubusercontent.com/3540544/112532515-31d58b80-8da9-11eb-84f9-aeca042edbc2.png)

Thanks for a great application.

I forgot, here are my specs
![image](https://user-images.githubusercontent.com/3540544/112536908-54b66e80-8dae-11eb-8397-6471b73edd42.png)

**Comment by matryer** [comment]
@leaanthony Do we need to similar work that we did to the tray to the submenu items?

**Comment by leaanthony** [comment]
Thanks for raising this @gson88. I'll check, but it looks like it might be skipping submenu nodes for the stylised text. It would be an easy fix if it is, and I'm 99% sure that's the issue.

**Comment by leaanthony** [comment]
@gson88 - was as expected. Fix done. Will be in the next xbar release. Here's a demo partly using your output:
![image](https://user-images.githubusercontent.com/1943904/112614372-4bdca000-8e75-11eb-9c83-09ff391cdf4e.png)

**Comment by gson88** [comment]
@leaanthony Nice. Good work

**Comment by jebabin** [comment]
Hello, is the fix really in v2.0.34-beta as in the relase note ? Even after upgrading I still have this issue, my example and the result bellow

Test
---
abcd | image=iVBORw0KGgoAAAANSUhEUgAAACIAAAAiAQMAAAAAiZmBAAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAFZaVRYdFhNTDpjb20uYWRvYmUueG1wAAAAAAA8eDp4bXBtZXRhIHhtbG5zOng9ImFkb2JlOm5zOm1ldGEvIiB4OnhtcHRrPSJYTVAgQ29yZSA2LjAuMCI+CiAgIDxyZGY6UkRGIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyI+CiAgICAgIDxyZGY6RGVzY3JpcHRpb24gcmRmOmFib3V0PSIiCiAgICAgICAgICAgIHhtbG5zOnRpZmY9Imh0dHA6Ly9ucy5hZG9iZS5jb20vdGlmZi8xLjAvIj4KICAgICAgICAgPHRpZmY6T3JpZW50YXRpb24+MTwvdGlmZjpPcmllbnRhdGlvbj4KICAgICAgPC9yZGY6RGVzY3JpcHRpb24+CiAgIDwvcmRmOlJERj4KPC94OnhtcG1ldGE+Chle4QcAAAAJcEhZcwAACxMAAAsTAQCanBgAAAADUExURf8AABniCTcAAAALSURBVAgdYxhGAAAAzAABBmNdUQAAAABJRU5ErkJggg==
--dummy
def | color=red 
--dummy
ghi | size="10"
--dummy

![Screenshot 2021-03-28 at 18 05 46](https://user-images.githubusercontent.com/11474713/112758771-4f486680-8ff0-11eb-95b3-180a182567df.png)

**Comment by leaanthony** [comment]
It wasn't in the last release - there was a mix up with branches. I've got a PR to merge the branches and can confirm the script you pasted shows this:
![image](https://user-images.githubusercontent.com/1943904/112893584-96d10e80-9126-11eb-9377-2f90c99f2264.png)
